### DISCUSSION TITLE
Large Chapters not working when using Media Mod on GoPro Hero8 Black

### DISCUSSION THREAD

**Discussion by looseopinions** [gopro/labs#376] [Q&A]
I installed Labs and enabled the 64bt Large Chapter feature. It works fine, unless I put my camera in the Media Mod housing. When in the housing and I record a video larger than 4GB, it just chapters it as 4GB files as it did before I installed the mod.

Is this a known issue or am I doing something wrong?

**Comment by dnewman-gpsw** [comment]
addressed in issues.
